docs/infrastructure/index.asciidoc

```diff
@@ -59,6 +59,18 @@ changes performed via Configure source are specific to that space. You can
 therefore easily make different subsets of the data available by creating
 multiple spaces with different data source configurations.
 
+[float]
+[[infra-read-only-access]]
+=== Read only access
+When you have insufficient privileges to change source configuration, the following
```

Comment:
Wouldn't it have to be either "**the** source configuration" or "source configuration**s**"? Same for the other occurrences below.

Reply:
Good point, thanks!